source/sass/main.scss

```diff
@@ -428,6 +428,7 @@ button {
 
     &:hover {
       border: 2px solid darken($nsf-yellow, 10%);
```

Comment:
Let's change this to `border-color: darken($nsf-yellow, 10%);` since we are just updating color.